src/proto/ast.proto

```diff
@@ -376,3 +463,25 @@ message PositionInfo {
 	required int32 end_line = 5;
 	required int32 end_col = 6;
 }
+
+/** A single declaration for specification */
+message SpecDeclaration {
+	repeated Modifier modifiers = 1;
+	repeated Statement specs = 2;
+}
+
+/** A single method for specification */
+message SpecMethod {
+	repeated SpecCase case = 1;
```

Comment:
this should be named 'cases' plural as it is repeated - also update Ast.java and SpecMethodProtoTuple.java